content/applications/sales/sales/products_prices/products/product_images.rst

```diff
@@ -1,130 +1,168 @@
 
 .. _product_images/setup-in-odoo:
 
 Odoo
 ----
 
-#. Go to :menuselection:`Settings --> General Settings --> Integrations`,
-   activate **Google Images** and save.
+#. In the Odoo database, go to :menuselection:`Settings app` and scroll to the
+   :guilabel:`Integrations` section. From here, check the box beside :guilabel:`Google Images`.
+   Then, click :guilabel:`Save`.
 
-#. Go back to :menuselection:`Settings --> General Settings--> Integrations`, enter your **API Key**
-   and **Search Engine ID** in **Google Images** settings and save again.
+   .. image:: product_images/google-images-setting.png
+      :align: center
+      :alt: The Google Images setting in the Odoo Settings app page.
+
+#. Next, re-enter the :menuselection:`Settings app`, and scroll to the :guilabel:`Integrations`
+   section. Then, enter the :guilabel:`API Key` and :guilabel:`Search Engine ID` in the fields
+   beneath the :guilabel:`Google Images` feature.
+
+#. Click :guilabel:`Save`.
 
 .. _product_images/get-product-images:
 
-Automatically get your product images in Odoo
-=============================================
+Product images in Odoo with Google Custom Search API
+====================================================
 
-The action to automatically get your product images in Odoo appears in any Products or Product
-Variants list view. Here is a step-by-step guide from the Inventory app.
+Adding images to products in Odoo can be done on any product or product variants. This process can
+be completed in any Odoo application that provides access to those pages (e.g. *Sales* app,
+*Inventory* app, etc.).
 
-#. Go to the Products menu (:menuselection:`Products --> Products` or :menuselection:`Products -->
-   Product Variants`) from any application that uses products like Inventory or Sales.
+Below is a step-by-step guide detailing how to utilize *Google Custom Search API* for automatic
+product images in Odoo using the Odoo *Sales* application:
 
-#. On the list view, select the products that needs an image.
+#. Navigate to the :guilabel:`Products` page in the *Sales* app (:menuselection:`Sales app -->
+   Products --> Products`. Or, navigate to the :guilabel:`Product Variants` page in the *Sales* app
+   (:menuselection:`Sales app --> Products --> Product Variants`).
 
-   .. important::
-      Only the 10,000 first selected products or product variants will be processed.
+#. Select the desired product that needs an image.
 
    .. note::
-      - Only the products or product variants with a barcode and without an image will be processed.
-      - If you select a product that has one or more variants from the Products view, each variant
-        matching the previous criteria will be processed.
+      Only the products, or product variants, with a barcode and do **not** an image are processed.
+
+      If a product selected with one or more variants, each variant matching the previous criteria
+      is processed.
```

Comment:
I think both of these sentences need to be rewritten. They don't make sense as is.